BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -229,6 +229,32 @@ namespace CGAL {
       > ::type  type;
   };
 
+    
+    template<typename NamedParameters>
+    class GetSplitVisitor
+    {
+    public:
+      struct DummySplitVisitor
+      {
+        template <typename T>
+        void start(const T&) const
+        {}
+        
+        template <typename T>
+        void operator()(const T&) const 
+        {}
+        
+        void done() const
```

Comment:
`end()` to be consistent with `start()`?

Reply:
then, should it be `begin()` and `end()`?